src/modules/sign/screens/DetailsMessage.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright 2015-2020 Parity Technologies (UK) Ltd.
```

Comment:
This file is not used at all right now, I realized I don't know exactly what to put here so it's not called from anywhere yet, but removing it is a bit painful while I hope to include it in final release (and start using it very soon) - let's leave it for now?

Reply:
Yea I would leave it for now, I think that is totally fine.